### DISCUSSION TITLE
Change target data type

### DISCUSSION THREAD

**Discussion by Fefe9487** [pycaret/pycaret#2894] [Q&A] [answered]
Hello,
I want to change the data type of the target (Label->Numeric), but i can't use numeric_feature this method.
What should I do to solve this problem?
p.s.The target data is a continuous variable (sholud use numeric), but pycaret judges it as label tpye.
![image](https://user-images.githubusercontent.com/65411889/185795627-9b4aecb8-9b42-4872-9635-a98d6d94e8a2.png)

**Comment by celestinoxp** [comment, marked as answer]
Label it means is the target.
You should define what kind type is your problem. classification or regression.
For classification before the setup(...), you should use from pycaret.classification import *
else, if is a regression type, you should use pycaret.regression import *

you can see pycaret documentation here: https://pycaret.gitbook.io/docs/
